src/cxy_ctl.erl

```diff
@@ -601,9 +601,22 @@ execute_wrapper(Mod, Fun, Args, Task_Type, Max_History, Start, Spawn_Or_Inline,
     end.
 
 -spec fail_wrapper(spawn | inline, any(), any()) -> no_return().
-fail_wrapper(spawn,  Call_Data, Stacktrace) -> erlang:error(spawn_failure,  [Call_Data, Stacktrace]);
-fail_wrapper(inline, Call_Data, Stacktrace) -> exit       ({inline_failure, [Call_Data, Stacktrace]}).
-
+fail_wrapper(spawn,  Call_Data, Stacktrace) -> erlang:error(spawn_failure,  [sanitize(Call_Data), Stacktrace]);
+fail_wrapper(inline, Call_Data, Stacktrace) -> exit       ({inline_failure, [sanitize(Call_Data), Stacktrace]}).
+
+% timeout on nested gen_server:call/2,3 - most callers are
+% gcache modules trying to replenish cache (create_key_value)
+% in the deepest callstack, redis connection goes away for a bit
+
+% search for 'spawn_failure' or 'inline_failure' in the logs, you will see more
+% pattern match for specific case - trade off between
+% valuable debugging information vs. private customer's data
+sanitize({exit, {timeout, {gen_server, call, _}},
+    {creating_new_value_with, gcache_auth, {check_auth, _}}}) ->
+    gcache_auth_timeout;
```

Comment:
"gcache_auth_timeout" seems to be less informative. Can we just remove credentials, e.g api key, from the log and keep other information?

Reply:
@dvliman - yeah, I thought that this is what we wanted to do actually and not just remove the entire payload... no?

Reply:
@anderswei @TT-lwang @nitzanharel updated per your comments: https://github.com/tigertext/epocxy/pull/18/commits/f90fa0c13dfb74c25132275f46efd9d7db0a1bab